backend/shop/urls.py

```diff
@@ -10,7 +14,9 @@
     path("role/new/", views.role_create, name="role_create"),
     path("permission/", views.permission_get, name="permission_get"),
     path("permission-user/", views.permission_user_get, name="permission_user"),
-    path("product/", views.product_get, name="product_get_all"),
+    path("product/", views.product_get, name="product_get_many"),
```

Comment:
To me get many doesn't tell me anything - it's confusing (why many, why not all, how do I tell how many)? Instead think of a model of -> "product_list" and then you can have "product_list" with a filter applied. More of a pipeline approach https://en.wikipedia.org/wiki/Pipeline_(computing)